environments/Test-Laptop.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "Test-Laptop",
   "override_attributes": {
+   },
+  "default_attributes": {
```

Comment:
As per the comments on issue #61, this can not be merged successfully as is; feel free to remove it for the remaining issues to be merged.